org/mozilla/jss/JSSProvider.java

```diff
@@ -59,6 +59,21 @@ public JSSProvider() {
         put("Alg.Alias.Signature.SHA256/RSA", "SHA-256/RSA");
         put("Alg.Alias.Signature.SHA256withRSA", "SHA-256/RSA");
 
+        put("Signature.SHA-256/RSA/PSS",
```

Comment:
I'm fine with these names for clarity and specifying which hash algorithm to use, but please add the supported name too, and allow for detection of the signature parameters:

(from `SunRsaSignEntries.java` in JDK11)
```java
        map.put("Signature.RSASSA-PSS",
                "sun.security.rsa.RSAPSSSignature");
```

And also the other aliases they expect:

```
src/java.base/share/classes/sun/security/rsa/SunRsaSignEntries.java:71:        map.put("Signature.RSASSA-PSS",
src/java.base/share/classes/sun/security/rsa/SunRsaSignEntries.java:129:        map.put("Alg.Alias.Signature.1.2.840.113549.1.1.10",     "RSASSA-PSS");
src/java.base/share/classes/sun/security/rsa/SunRsaSignEntries.java:130:        map.put("Alg.Alias.Signature.OID.1.2.840.113549.1.1.10", "RSASSA-PSS");

```

Reply:
Done.